frontend/src/views/ApplicationRegister.vue

```diff
@@ -142,13 +181,58 @@ export default {
         ...recruitmentItem,
         contents: "",
       }))
-      this.url = ""
+      this.referenceUrl = ""
     },
-    save() {
-      // TODO: 임시 저장 기능 추가 필요
+    async save(isSubmitted) {
+      let applicationForm = this.parseApplicationInfo()
+      applicationForm = {
+        ...applicationForm,
+        isSubmitted: isSubmitted,
+      }
+      return await this.saveOrUpdate(applicationForm)
```

Comment:
```suggestion
      const applicationForm = {
        ...this.parseApplicationInfo(),
        isSubmitted: isSubmitted,
      }
      return this.saveOrUpdate(applicationForm)
```
[핀의 리뷰](https://github.com/woowacourse/service-apply/pull/113/files#r499339964)까지 반영하면 이런 느낌이겠네요